doc/04-Scanning.md

```diff
@@ -0,0 +1,85 @@
+# <a id="Scanning"></a>Scanning
+
+The Icinga Certificate Monitoring provides CLI commands to scan arbitrary **hosts** and **IPs** in various ways.
```

Comment:
Why arbitrary? Hosts and IPs have to be configured. I don't think that's arbitrary.